fdbserver/RESTKmsConnector.actor.cpp

```diff
@@ -774,14 +776,17 @@ Future<T> kmsRequestImpl(
 	// error (time-out or connection-failed), then continue enumeration. Otherwise, bubble up the error.
 	// Step-2: Refresh KmsUlrStore cached URLs by re-discovering KMS URLs and loop Step-1
 
-	state int pass = 0;
 	state KmsUrlCtx<KmsUrlPenaltyParams>* urlCtx;
+	state int pass = 0;
+	state double retryDelay = 0.05; // 50 ms
```

Comment:
Nit: we could make this a knob

Reply:
Sure, seems like a valid ask. Will have it as a followup PR. Nim also had similar comment.